tests/src/Kernel/Entity/GroupAudienceTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\GroupAudienceTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Og;
+use Drupal\Component\Utility\Unicode;
+
+/**
+ * @group og
+ */
+class GroupAudienceTest extends KernelTestBase {
+
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['entity_test', 'user', 'field', 'entity_reference', 'og', 'system'];
+
+  /**
+   * @var Array
+   *   Array with the bundle IDs.
```

Comment:
I don't think the comment should get indentation similar to [this](https://github.com/amitaibu/og/blob/8.x-1.x/src/Entity/OgMembership.php#L57)